templates/template_functions.go

```diff
@@ -292,3 +294,13 @@ func contains(arr interface{}, comp string) bool {
 	}
 	return false
 }
+
```

Comment:
In this file, line 35 you need to set Vars: `vars.Set("torrentFileExists", torrentFileExists)`

Reply:
totally forgot vars.set also needed to be done for functions and not just variables